aws/resource_aws_iot_topic_rule_test.go

```diff
@@ -343,16 +361,6 @@ resource "aws_iot_topic_rule" "rule" {
   sql = "SELECT * FROM 'topic/test'"
   sql_version = "2015-10-08"
 
-  // Fake data
-  dynamodb {
-    hash_key_field = "hash_key_field"
-    hash_key_value = "hash_key_value"
-    payload_field = "payload_field"
-    range_key_field = "range_key_field"
```

Comment:
😅 now we have the inverse problem where `range_key_field`, etc. is not being acceptance tested at all. Could you create a test and test configuration very similar to the new one which includes the newly optional arguments? See also `TestAccAWSIoTTopicRule_firehose_separator` which tests the optional argument separate from `TestAccAWSIoTTopicRule_firehose`. Thanks!